debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SQLServerNumericColumnIT.java

```diff
@@ -86,16 +89,20 @@ public void decimalModeConfigString() throws Exception {
         assertConnectorIsRunning();
         TestHelper.waitForSnapshotToBeCompleted();
 
-        connection.execute("INSERT INTO tablenuma VALUES (111.1111, 1111111, 1111111.1, 1111111 );");
-        final SourceRecords records = consumeRecordsByTopic(1);
-        final List<SourceRecord> tableA = records.recordsForTopic("server1.dbo.tablenuma");
-        Assertions.assertThat(tableA).hasSize(1);
-        final Struct valueA = (Struct) tableA.get(0).value();
-        assertSchema(valueA, Schema.OPTIONAL_STRING_SCHEMA);
-        Assertions.assertThat(((Struct) valueA.get("after")).get("cola")).isEqualTo("111.1111");
-        Assertions.assertThat(((Struct) valueA.get("after")).get("colb")).isEqualTo("1111111");
-        Assertions.assertThat(((Struct) valueA.get("after")).get("colc")).isEqualTo("1111111.1");
-        Assertions.assertThat(((Struct) valueA.get("after")).get("cold")).isEqualTo("1111111");
+        TestHelper.forEachDatabase(databaseName -> {
```

Comment:
Why is it all the test changes are needed?

Reply:
We reworked the existing tests to make sure all the original scenarios work with the new implementation/configuration. Each test now covers multiple databases.

What would be the preferred way of testing the changes? On the one hand, it makes sense to make as little changes as possible to the existing tests suite (ensures BC), on the other hand, we want all cases to be tested with the new configuration applied.